### PR TITLE
Correct mentions of windows defender smart screen

### PR DESCRIPTION
**Pull request by martyav**
As per conversation with Juli Hooper, we are updating in-text mentions and descriptions of SmartScreen to always include the full product name ("Windows Defender SmartScreen"), and removing "Filter" where appropriate. In some cases, ADMX files or other products' UI may use the old "SmartScreen Filter" name -- particularly problematic for paths. **We are currently waiting to hear back on whether it is safe to change the name when describing UI in certain products.**

### CODE REVIEW

File: windows/security/information-protection/secure-the-windows-10-boot-process.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ ms.author: dansimp
...
 
-Windows 10 has multiple levels of protection for desktop apps and data, too. Windows Defender uses signatures to detect and quarantine apps that are known to be malicious. The SmartScreen Filter warns users before allowing them to run an untrustworthy app, even if it’s recognized as malware. Before an app can change system settings, the user would have to grant the app administrative privileges by using User Account Control.
+Windows 10 has multiple levels of protection for desktop apps and data, too. Windows Defender uses signatures to detect and quarantine apps that are known to be malicious. The Windows Defender SmartScreen warns users before allowing them to run an untrustworthy app, even if it’s recognized as malware. Before an app can change system settings, the user would have to grant the app administrative privileges by using User Account Control.
```
Comment (appcompatguy) [inline windows/security/information-protection/secure-the-windows-10-boot-process.md]: "The Windows Defender SmartScreen warns users..." sounds awkward - shouldn't this just be "Windows Defender SmartScreen warns users..."?

File: windows/security/threat-protection/windows-defender-antivirus/configure-real-time-protection-windows-defender-antivirus.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ To configure these settings:
 Location | Setting | Description | Default setting (if not configured)
 ---|---|---|---
 Real-time protection | Monitor file and program activity on your computer | The Windows Defender Antivirus engine makes note of any file changes (file writes, such as moves, copies, or modifications) and general program activity (programs that are opened or running and that cause other programs to run) | Enabled
-Real-time protection | Scan all downloaded files and attachments | Downloaded files and attachments are automatically scanned. This operates in addition to the SmartScreen filter, which scans files before and during downloading | Enabled
+Real-time protection | Scan all downloaded files and attachments | Downloaded files and attachments are automatically scanned. This operates in addition to the Windows Defender SmartScreen, which scans files before and during downloading | Enabled
```
Comment (appcompatguy) [inline windows/security/threat-protection/windows-defender-antivirus/configure-real-time-protection-windows-defender-antivirus.md]: "in addition to the Windows Defender SmartScreen" sounds awkward - that's an extraneous "the"

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Windows Defender SmartScreen helps to protect your employees if they try to visi
 - Checking downloaded files against a list of files that are well known and downloaded by many Windows users. If the file isn't on that list, SmartScreen shows a warning, advising caution.
 
     >[!NOTE]
-    >Before Windows 10, version 1703 this feature was called the SmartScreen Filter when used within the browser and Windows SmartScreen when used outside of the browser.
+    >Before Windows 10, version 1703 this feature was called the Windows Defender SmartScreen when used within the browser and Windows SmartScreen when used outside of the browser.
```
Comment (appcompatguy) [inline windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]: This is not historically accurate. It was called the SmartScreen Filter. If we're going to have a note on history, we should note the original name, plus note the change we're actively making on Windows Defender SmartScreen.

**Comment by illfated** [comment]
**new-in-windows-mdm-enrollment-management.md** :
- public branch: last updated in commit 1b10d30 5 days ago
- master branch: last updated in commit d35e66c 14 days ago

**policy-configuration-service-provider.md** :
- public branch: last updated in commit 081564e yesterday
- master branch: last updated in commit c716d98 15 days ago

**policy-csp-update.md** :
- public branch: last updated in commit 83eae2a yesterday
- master branch: last updated in commit d35e66c 14 days ago

To me, it does not look like this has to do with the "latest version", but which version of the document takes precedence over the other.
I can only presume that there is some confusion regarding precedence between the [`master`](https://github.com/MicrosoftDocs/windows-itpro-docs/tree/master) branch and the [`public`](https://github.com/MicrosoftDocs/windows-itpro-docs/tree/public) branch.

**Comment by illfated** [comment]
Oh, OK. Fair enough, it seems my comment did make a wrong assumption there. Thank you for clarifying. I have probably been more confused than anyone else here, then.

File: windows/client-management/mdm/policy-csp-update.md [hunk 4]
```diff
@@ -195,6 +197,9 @@ manager: dansimp
   <dd>
     <a href="#update-setedurestart">Update/SetEDURestart</a>
   </dd>
+  <dd>
```
[the remark this answers is not in the document]
Comment (martyav) [reply windows/client-management/mdm/policy-csp-update.md]: I was a little confused at first, but I see now that I should have been updating via the public branch, not master!

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md (hunk 3 above)
Comment (martyav) [reply windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]: I corrected this. Thank you.

**Comment by martyav** [comment]
Apologies for leaving you hanging -- I just made two commits addressing the requested changes. I didn't realize when I created this PR that the most recent updates were in /public, not /master -- thank you @ManikaDhiman for alerting me!

**Comment by ManikaDhiman** [comment]
> Apologies for leaving you hanging -- I just made two commits addressing the requested changes. I didn't realize when I created this PR that the most recent updates were in /public, not /master -- thank you @ManikaDhiman for alerting me!

Thanks for the updates, @martyav! I approved changes in the MDM topics.

**Comment by e0i** [comment]
@Dansimp Would you have a suggestion to proceed with this PR as the time permits? Is it perhaps feasible to move forward with merge with the approval of two authors? Thank you for your time.

**Comment by illfated** [comment]
There are 2 occurrences of this sentence in the PR content:

2.  Verify the setting **Help protect me from malicious sites and download with Windows Defender SmartScreen** is disabled.

Technically speaking, the phrasing is correct, but I would like to suggest to make it slightly more fluent by adding the word "that":

2.  Verify that the setting **Help protect me from malicious sites and download with Windows Defender SmartScreen** is disabled.

**Pull request merged**